app/assets/javascripts/gaap-analytics.js

```diff
@@ -1,6 +1,6 @@
 // Taken from https://raw.githubusercontent.com/alphagov/gaap-analytics/master/build/gaap-analytics.js
```

Comment:
is this just the latest version of an upstream dependency?

Reply:
Yeah. This is the latest version which includes a few dependencies. One day we'll setup npm for the app so we can pull this package from npmjs.com but for now its a copy and paste job.